gateway/src/apicast/backend_client.lua

```diff
@@ -158,15 +158,21 @@ local function create_token_path(service_id)
 end
 
 -- Returns the authorize options that 3scale backend accepts. Those options
--- are specified via headers. Right now there are 2:
+-- are specified via headers:
 --  - rejection_reason_header: asks backend to return why a call is denied
 --    (limits exceeded, application key invalid, etc.)
---  - no_nody: when enabled, backend will not return a response body. The
+--  - no_body: when enabled, backend will not return a response body. The
 --    body has many information like metrics, limits, etc. This information is
 --    parsed only when using oauth. By enabling this option will save some work
 --    to the 3scale backend and reduce network traffic.
+--  - limit_headers: when enabled and the request is rate-limited, backend
+--    returns the number of seconds remaining until the limit expires. It
+--    returns -1 when there are no limits. With this header, backend returns
```

Comment:
@davidor https://tools.ietf.org/html/rfc7231#section-7.1.3 
```A delay-seconds value is a non-negative decimal integer, representing
   time in seconds.``` 

In https://github.com/3scale/apisonator/blob/master/docs/extensions.md#limit_headers-boolean I'd add a notice on that.